src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -472,10 +472,15 @@ def _supplementary_bundle(self, bundle_fqid: TDRAnvilBundleFQID) -> TDRAnvilBund
                 linked_file_refs.add(file_ref)
         dataset_ref, dataset_row = self._get_dataset(source)
         result.add_entity(dataset_ref, self._version, dict(dataset_row))
-        result.add_links([
-            EntityLink(inputs=singleton(dataset_ref),
-                       outputs=frozenset(linked_file_refs))
-        ])
+        # Avoid inserting "degenerate" links with an empty list of outputs, i.e.
```

Comment:
"degenerate" -> malformed ?

Reply:
Not a bad choice, but I think degenerate is more specific https://en.wikipedia.org/wiki/Degeneracy_(mathematics)